iris/src/iris/pipeline/tutor_suggestion/tutor_suggestion_pipeline.py

```
@@ -0,0 +1,294 @@
+        :return: Relevant lecture content as a string.
+        """
+        lecture_content = lecture_content_retrieval(dto, self.summary_text, self.db)
+        prompt = ChatPromptTemplate.from_messages(
+            [("system", lecture_summary_prompt())]
+        )
+        try:
+            completion_args = CompletionArguments(temperature=0, max_tokens=8000)
+            if self.variant == "advanced":
+                model = "gpt-4.1"
+            else:
+                model = "gpt-4.1-mini"
```

Ruff (0.12.2)

Use ternary operator `model = "gpt-4.1" if self.variant == "advanced" else "gpt-4.1-mini"` instead of `if`-`else`-block

Replace `if`-`else`-block with `model = "gpt-4.1" if self.variant == "advanced" else "gpt-4.1-mini"`

(SIM108)